src/classes/Forms.ts

```diff
@@ -792,17 +792,16 @@ export default class Forms extends OneBlinkAPI {
    * #### Example
    *
    * ```javascript
-   *
    * forms
    *   .migrateForm({
-   *      formsAppEnvironmentId: 2,
-   *      sourceFormId: 123,
-   *      targetFormId: 234,
-   *      elements: true,
-   *      submissionEvents: false,
-   *      serverValidation: false,
-   *      externalIdGeneration: false,
-   *      postSubmissionAction: false,
+   *     formsAppEnvironmentId: 2,
+   *     sourceFormId: 123,
+   *     targetFormId: 234,
+   *     elements: true,
+   *     workflowEvents: false,
```

Comment:
We don't actually want to change the name of this property, because it won't work. Just the description for it.